### PR TITLE
isort to sort imports alphabetically

### PR DESCRIPTION
**Pull request by erikwijmans**
## Motivation and Context

We like pretty code, isort makes code prettier, thus we use isort.  I also added isort to the CI to keep things pretty.

`isort -rc .`
`black --line-length 79 .`

## How Has This Been Tested

Just re-orders imports! 

Although I choose the isort configs to make isort and black play nice, so I may have overfit to my test-set :)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,14 +4,22 @@
...
 import os.path
 import sys
 
 import setuptools
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "habitat"))
+
 from version import VERSION  # noqa
 
+
```
Comment (abhiskk) [inline setup.py]: Is BLACK okay with this, this seems like a PEP violiation

File: habitat_baselines/agents/slam_agents.py [hunk 2]
```diff
@@ -1,39 +1,38 @@
+# https://sumit-ghosh.com/articles/python-download-progress-bar/
```
Comment (abhiskk) [inline habitat_baselines/agents/slam_agents.py]: Is this getting used somewhere?

Also I just noticed that this file is missing the facebook header, I can add it in another PR.

File: setup.py (hunk 1 above)
Comment (erikwijmans) [reply setup.py]: Seems like it didn't run on setup.py for some reason....

File: habitat_baselines/agents/slam_agents.py (hunk 2 above)
Comment (erikwijmans) [reply habitat_baselines/agents/slam_agents.py]: Looks like that comment is referencing this function: https://github.com/facebookresearch/habitat-api/blob/master/habitat_baselines/agents/slam_agents.py#L39

Moved it to there

**Comment by erikwijmans** [comment]
I'm currently trying to figure out why isort is failing with travis.  It works locally.....

**Comment by erikwijmans** [comment]
It's different versions of isort :/

**Comment by erikwijmans** [comment]
@mathfac I believe this was done with #97?

**Comment by mathfac** [comment]
@erikwijmans, right.

**Comment by erikwijmans** [comment]
Closing then!

**Pull request closed without merging**